build-utils/css/README.md

```diff
@@ -1,8 +1,112 @@
 # **Build styles**
-This script is primarily created for generating accessible style constants and css file from configured style constant to benefit from strongly typed css variable for Amino consumers.  
+This script is primarily created for generating accessible style constants and css file from configured style constant to benefit from strongly typed css variable and jsdocs support for Amino consumers.   
   
-Main script for this building process is `build-utils/css/buildTheme.ts`.
+- Sub script for generating dynamic constants to be consumed in `build-utils/css/constants/theme.ts` process is `build-utils/css/buildLogicConstants.ts`
+- Main script for this whole building process is `build-utils/css/buildTheme.ts`.
+---
+
+## **Generating dynamic constant process overview**
+Sub script `build-utils/css/buildLogicConstants.ts` is primarily created for having a way to generate a constant programmatically with logic and customized jsdocs comments.  
+**REASON**: Typescript will not read and provide jsdocs info on the spread operators (Ex: `...sthing`), so we can't benefit from jsdocs tag when using typescript key suggestion.  
+  
+### **When you run the script, it will**:
+*BEBFORE STARTED, UNTIL WE HAVE A SCRIPT TO GENERATE THE TEMPLATE, YOU MAY NEED TO MANUALLY PUT A FILE WITH SOME REQUIRED FUNCTIONS AND VARIABLES WITH NAMING SYNTAX `_${name}.ts` (WHICH WILL BE EXPLAINED HOW DEEPER BELOW) INTO `build-utils/css/constants/logics`.*  
```

Comment:
This really looks great Thein 🎉   I think there might be a tiny typo on `*BEFORE`?

Reply:
Good catch 😆 Thanks for reading and noticing that 😉